zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/incident/IncidentResolveProcessor.java

```diff
@@ -29,6 +29,8 @@
 
 public final class IncidentResolveProcessor implements TypedRecordProcessor<IncidentRecord> {
 
+  public static final String NO_RETRIES_LEFT_MSG =
```

Comment:
🔧 Let's instruct api users how to solve the issue. At the end of the error message, let's mention that they can update the job retries and retry resolving the incident.